content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -125,13 +122,16 @@ This will decrease the amount data flowing through the system, which will in tur
     caption="P90 latency vs client connections"
 >}}
 
+{{< image width="90%"
+    link="latency_p99.svg"
+    alt="P99 latency vs client connections"
+    caption="P99 latency vs client connections"
+>}}
+
 - `baseline` Client pod directly calls the server pod, no sidecars are present.
-- `server-sidecar` Only server sidecar is present.
 - `both-sidecars` Client and server sidecars are present. This is a default case inside the mesh.
-- `nomixer-both` Same as **both-sidecars** without Mixer.
-- `nomixer-server` Same as **server-sidecar** without Mixer.
+- `none-both` Istio proxy with no Istio specific filters configured.
 - `telemetryv2-nullvm_both` Same as **both-sidecars** but with telemetry v2. This is targeted to perform the same as "No Mixer" in the future.
```

Comment:
Unclear what "No Mixer" means here, since there are no more `nomixer-*` configs.

Reply:
Actually I think a more general problem is that the 4 config names listed here are not the same as the names in the diagrams. I think they should be the same.

Reply:
thanks for pointing this out, let me fix it.

Reply:
@frankbu what about the current version?

Reply:
cc @mandarjog